.github/workflows/mint-release.yml

```diff
@@ -61,10 +57,13 @@ jobs:
     - name: Tag the release
       run: |
         git tag "v${{ github.event.inputs.tag }}" -m "Version v${{ github.event.inputs.tag }}"
-        git tag
 
-    - name: Push tag, branch state
+    - name: Create or checkout release branch
+      run: |
+        BRANCH=$(echo $VERSION | awk 'BEGIN{FS=OFS="."} {$3="x"} 1' | cut -f1,2,3 -d'.' | awk -F'rc' '{print $1 ORS $2}')
+        git checkout $BRANCH 2>/dev/null || git checkout -b $BRANCH
+
+    - name: Push tag and branch to remote repository
```

Comment:
🛗 Moved from above, where the only change is extracting `$VERSION` so it's defined globally.